source/bronze/tests/migrations/test_migrations.py

```diff
@@ -1,25 +1,25 @@
 import testcommon.dataframes.assert_schemas as assert_schemas
 from pyspark.sql import SparkSession
 
-from opengeh_bronze.domain.constants.database_names import DatabaseNames
 from opengeh_bronze.domain.constants.table_names import TableNames
 from opengeh_bronze.domain.schemas.bronze_measurements import (
     bronze_measurements_schema,
 )
 from opengeh_bronze.domain.schemas.submitted_transactions import (
     submitted_transactions_schema,
 )
+from opengeh_bronze.infrastructure.settings import CatalogSettings
+
+database_name = CatalogSettings().bronze_database_name  # type: ignore
```

Comment:
```suggestion
bronze_database_name = CatalogSettings().bronze_database_name  # type: ignore
```

remember to edit the use